docs/code/storage_config.py

```diff
@@ -60,5 +60,5 @@
 """"""
 
 """Create authorized local storage dataset"""
-dataset_client = gas.create_dataset("dataset_name", config_name="local_storage_config")
+dataset_client = gas.create_dataset("<DATASET_NAME>", config_name="<LOCAL_STORAGE_CONFIG>")
```

Comment:
Here, the config name should be the same as the name in `create_local_storage_config` before.